src/main.rs

```diff
@@ -433,7 +433,6 @@ mod tests {
     }
 
     #[test]
-    #[ignore]
```

Comment:
Could you please squash this commit into the previous one?

Reply:
Sure!